publication-commons/src/main/java/no/unit/nva/PublicationHandler.java

```diff
@@ -89,10 +89,11 @@ public static ObjectMapper createObjectMapper() {
         return new ObjectMapper()
                 .registerModule(new ProblemModule())
                 .registerModule(new JavaTimeModule())
-                .configure(SerializationFeature.WRITE_DATES_AS_TIMESTAMPS, false)
-                .configure(SerializationFeature.INDENT_OUTPUT, true)
-                .setSerializationInclusion(JsonInclude.Include.NON_NULL)
-                .configure(DeserializationFeature.FAIL_ON_UNKNOWN_PROPERTIES, false);
+                .enable(DeserializationFeature.ACCEPT_EMPTY_STRING_AS_NULL_OBJECT)
+                .enable(SerializationFeature.INDENT_OUTPUT)
+                .disable(DeserializationFeature.FAIL_ON_UNKNOWN_PROPERTIES)
```

Comment:
I think for this one, it is better if the author of each Java bean decides himself about this with the tag `@JsonIgnoreProperties(ignoreUnknown = true)` but I do not have strong opinions about it.

Reply:
Yes, we can do it like that when you create the new lib